server/data.py

```diff
@@ -139,53 +139,27 @@ def _get_het_hom_index(self, data, index, use_gt, gp_thres, calc_info):
                 gt = s[0]
                 if gt == '1|1' or gt == '1/1':
                     hom_alt_i.append(i)
-                elif not (gt == '0|0' or gt == '0/0'):
+                elif (gt == '1|0' or gt == '1/0' or gt == '0|1' or gt == '0/1'):
                     het_i.append(i)
+                elif (gt == '0|0' or gt =='0/0'):
+                    wt_hom_i.append(i)
+                else:
+                    # if there are missing calls in the raw chip data
+                    missing_i.append(i)
```

Comment:
now anything missed be the previous if else logic would be counted as missing... true in this case but much better is to check that the gt is one of the expected ones and error out if unknown coding is present.